v0.59.0 added support for arm

nmcc1212 opened the pull request:
update to match https://github.com/MicrosoftDocs/windows-uwp/pull/3899

nmcc1212 committed:
Update index.md

update to match https://github.com/MicrosoftDocs/windows-uwp/pull/3899

PRMerger3 commented:
@nmcc1212 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Court72 commented:
@mattwojo

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

Jay-o-Way commented on hub/powertoys/index.md:
```diff
@@ -18,8 +18,7 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 - **x64**: Supported
-- **Arm**: In development (see [issue #490](https://github.com/microsoft/PowerToys/issues/490))
-
+- **Arm**: Supported
```
```suggestion
- **ARM**: Supported
```

Jay-o-Way reviewed (commented):
Might as well make it uppercase

nmcc1212 committed:
Update hub/powertoys/index.md

Co-authored-by: Jay <65828559+Jay-o-Way@users.noreply.github.com>

PRMerger17 commented:
@nmcc1212 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

mattwojo committed:
correct casing

There's an agreement with the ARM organization for us to use "Arm" in the docs.

mattwojo commented:
> Might as well make it uppercase

We have an agreement with the ARM organization to use "Arm" in our docs.
